apps/voting/vite.config.ts

```diff
@@ -1,22 +1,29 @@
-import { defineConfig } from 'vite';
+import { PluginOption, UserConfig, defineConfig } from 'vite';
 import react from '@vitejs/plugin-react-swc';
 import tsconfigPaths from 'vite-tsconfig-paths';
 import wasm from 'vite-plugin-wasm';
 import topLevelAwait from 'vite-plugin-top-level-await';
+import handlebars from 'vite-plugin-handlebars';
+import { v4 as uuid } from 'uuid';
 import 'dotenv/config';
 
 const DEFAULT_NETWORK = 'testnet';
 const DEFAULT_NODE = 'http://localhost:20001';
-const DEFAULT_BACKEND_API = 'http://localhost:8080';
+const DEFAULT_BACKEND_API = ''; // App is served from the same URL as the API.
 
 /**
  * Validates environment variable present at `envField` as a URL.
  *
  * @param envField - The name of the environment variable to validate
  * @throws If environment variable is deemed an invalid URL.
  */
-function validateURL(envField: string): void {
+function validateURL(envField: string, allowUnset = true): void {
     const urlCandidate = process.env[envField];
+    if (!allowUnset && !urlCandidate) {
+        console.log(envField, urlCandidate);
```

Comment:
```suggestion
        console.error(envField, urlCandidate);
```

Reply:
Removed it instead, as the error is shown in the terminal anyway.